qiskit/transpiler/runningpassmanager.py

```diff
@@ -311,42 +322,76 @@ class FlowControllerLinear(FlowController):
     def __init__(self, passes, options):  # pylint: disable=super-init-not-called
         self.passes = self._passes = passes
         self.options = options
+        self.property_set = None
+
+    def do_passes(self, pass_manager, dag, property_set):
```

Comment:
this method seems identical to the base class method, so i think it should be removed.

The same is true I think for the the variables `self.passes`, `self.options`, `self.property_set` which get repeated in all the single `FlowControllers` but are all the same as the base class.

Reply:
good catch! Fixed in bface9dae. The __init__ case is different. I agree that probably needs a refactor. Task for a following PR.